### PR TITLE
Task6

### PR DESCRIPTION
**Pull request by iharyachnik**

### CODE REVIEW

File: models/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var fs        = require('fs');
```
Comment (DraXu3) [inline models/index.js:3]: Why do you use obsolete `var`'s here?

File: models/products.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var products = sequelize.define('products', {
```
Comment (DraXu3) [inline models/products.js:3]: ...and here

File: models/users.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var users = sequelize.define('users', {
```
Comment (DraXu3) [inline models/users.js:3]: ...and also here

**Review by DraXu3 — CHANGES_REQUESTED** [review]
Use modern (actually, which should be already regular) ES2015 syntax wherever it's possible

File: models/index.js (hunk 1 above)
Comment (iharyachnik) [reply models/index.js:3]: It happens because of using sequelize-cli. This files (models, seeds, migrations) are automatic-generated.

**Pull request merged**